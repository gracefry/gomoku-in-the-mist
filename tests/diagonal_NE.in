place J15
place A1
place K14
place A2
place L13
place A3
place M12
place A4
place N11
