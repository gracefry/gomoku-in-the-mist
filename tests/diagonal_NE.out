Black wins!
J15A1K14A2L13A3M12A4N11
Thank you for playing!
